src/styles/base.scss

```diff
@@ -1555,3 +1555,592 @@ td, th {
+}
+
+i.icon {
+  font-size: 15px;
+}
+
+.side-nav li>a>i {
+  margin: 0px;
+}
+
+a {
+  transition: all 0.2s ease-in-out;
+}
+
+a.white-link {
+  color: $light-link;
+}
+
+a.white-link:hover {
+  color: $link-hover-dark;
+}
+
+a.highlight-link {
+  color: $highlight;
+}
+
+a.light-link {
+  color: $med-link;
+  &:hover {
+    color: $highlight;
+  }
+}
+
+a.med-link {
+  color: $med-link;
+}
+
+a.med-link:hover {
+  color: $dark-link;
+}
+
+a.med-black-link {
+  color: $med-black;
+}
+
+a.med-black-link:hover {
+  color: $highlight;
+}
+
+a.dark-link {
+  color: $dark-link;
+}
+
+a.text-light-black:hover {
+  color: $highlight;
+}
+
+/*to remove gutter from col*/
+
+.rm-gut {
+  margin-left: -10px !important;
+  margin-right: -10px !important;
+}
+
+.form-icon {
+  i {
+    position: absolute;
+    right: 0;
+    top: 15px;
+    color: $light-gray;
+    font-size: 0.9em;
+    transition: all 0.2s ease-in-out;
+  }
+}
+
+/* label color */
+
+.input-field label,
+input[type=checkbox]+label {
+  color: $light-gray;
+  font-size: 0.8em;
+  font-weight: 400;
+}
+
+input[type="checkbox"].filled-in:checked+label:after {
+  width: 17px;
+  height: 17px;
+  border: 1px solid $link-hover-dark;
+  background-color: $link-hover-dark;
+}
+
+input[type="checkbox"]+label {
+  padding-left: 24px;
+  line-height: 20px;
+}
+
+input[type="checkbox"].filled-in:checked+label:before {
+  top: 0;
+  left: 2px;
+  width: 5px;
+  height: 11px;
+}
+
+input[type="checkbox"].filled-in:not(:checked)+label:after {
+  width: 17px;
+  height: 17px;
+  border: 1px solid $light-gray;
+  background-color: transparent;
+}
+
+/* label focus color */
+
+.input-field input[type=email]:focus+label,
+.input-field input[type=password]:focus+label,
+.input-field input[type=text]:focus+label {
+  color: $light-gray;
+}
+
+/* label underline focus color */
+
+.input-field input[type=email]:focus,
+.input-field input[type=password]:focus,
+.input-field input[type=text]:focus {
+  border-bottom: 1px solid $link-hover-dark;
+  box-shadow: 0 0px 0 0 $link-hover-dark;
+}
+
+/* valid color */
+
+.input-field input[type=email].valid,
+.input-field input[type=password].valid,
+.input-field input[type=text].valid {
+  border-bottom: 1px solid $link-hover-dark;
+  box-shadow: 0 0px 0 0 #000;
+}
+
+/* invalid color */
+
+.input-field input[type=email].invalid,
+.input-field input[type=password].invalid,
+.input-field input[type=text].invalid {
+  border-bottom: 1px solid $link-hover-dark;
+  box-shadow: 0 0px 0 0 #000;
+}
+
+/* icon prefix focus color */
+
+input {
+  font-weight: 300;
+  border-bottom: 1px solid $light-gray;
+}
+
+.input-field input[type=email]:focus+.form-icon,
+.input-field input[type=password]:focus+.form-icon,
+.input-field input[type=text]:focus+.form-icon {
+  i {
+    color: $light-link;
+  }
+}
+
+.input-field input[type=email]:focus+.form-icon-dark,
+.input-field input[type=password]:focus+.form-icon-dark,
+.input-field input[type=text]:focus+.form-icon-dark {
```

Comment:
I checked my code and I am using these styles as I have some customized input elements.

Reply:
I think at all the places we are having the same input containing same css and even if you want to customize it then it would be better if you could do it in the existing one i.e. inside input component scss file.

Reply:
Let's keep it like that now. we can modify into that later.